### ISSUE TITLE
Lab Module

### ISSUE THREAD

**Issue by piratedkhan** [earthians/marley#22]
I have suggestion that Laboratory Module should be independent of Healthcare Module. its because of its nature medical labs do exists without the complete healthcare system. Some good software for healthcare like GNU Health provide independent software  for lab management.  
some features which i would like to see is.

- raw material management .
- online results mechanism for patients to see their results online.
- sms/email notification for patient/customer upon awaited result.

**Comment by ChillarAnand** [comment]
> I have suggestion that Laboratory Module should be independent of Healthcare Module

Agreed. There are many diagnostic centers that just need laboratory module but not the entire healthcare module.

In that case, they can just use the lab features and other required features. Based on user roles, we can limit the interface that is accessible to the users. 


> raw material management 

This can be managed with stock module.

> online results mechanism for patients to see their results online.

We need to build a patient portal for this. It is in our roadmap.

> sms/email notification for patient/customer upon awaited result.

We already have [this feature](https://docs.erpnext.com/docs/v13/user/manual/en/healthcare/healthcare_settings#7-laboratory-sms-alerts).
